README.md

```diff
@@ -89,6 +89,7 @@ Because of the way external configuration has been implemented in various .NET f
 | --- | --- | --- |  --- |
 | .NET Framework 4.6.2+ | `net462` | app or library | _System.Configuration_ |
 | .NET Framework 4.6.2+ | `net462` | app or library | _Microsoft.Extensions.Configuration_ |
+| .NET Framework 4.7.2+ | `net472` | app or library | _Azure.Identity_ |
```

Comment:
This is not very clear. Please add a chapter (e.g. named "Azure Identity") right before "Deprecated Features" and explain the feature and also mention on which target frameworks it is supported (.NET Core 3.1+, .NET Framework 4.7.2+).

Reply:
I added a paragraph at the end as requested